src/openrct2-ui/windows/Dropdown.cpp

```diff
@@ -22,8 +22,8 @@ constexpr int32_t DROPDOWN_TEXT_MAX_ROWS = 32;
 
 constexpr int32_t DROPDOWN_ITEM_HEIGHT = 12;
 
-int32_t gAppropriateImageDropdownItemsPerRow[] = {
-    1, 1, 1, 1, 2, 2, 3, 3, 4, 3, 5, 4, 4, 5, 5, 5, 4, 5, 6, 5, 5, 7, 4, 5, 6, 5, 6, 6, 6, 6, 6, 8, 8, 0,
+static int32_t _appropriateImageDropdownItemsPerRow[34] = {
```

Comment:
Can be made const as well, can't it?

Could probably use an std::array instead, as well.

Reply:
Is there a point to using std::array for a constant array, anyway?

Reply:
Well you wouldnt need to use `std::size` as you can use `.size()`. I don't really see much point changing it unless you want to iterate over and use the ranged for loop.

Reply:
You can use ranged for loops for built-in arrays: https://gcc.godbolt.org/z/TVioqG

No need to use std::array containers when that's your goal.

Reply:
I think it makes the code more like C++

Reply:
I think it makes the code more like C++